packages/vite-plugin-kit-routes/src/lib/format.ts

```diff
@@ -19,18 +19,34 @@ export const format = (margin: { left?: number; top?: number; bottom?: number },
   )
 }
 
-export const appendSp = `/**
+export const appendSp = `type ParamValue = string | number | undefined
+
+/**
  * Append search params to a string
  */
-const appendSp = (sp?: Record<string, string | number | undefined>, prefix: '?' | '&' = '?') => {
+const appendSp = (sp?: Record<string, ParamValue | ParamValue[]>, prefix: '?' | '&' = '?') => {
```

Comment:
Do you think `appendSp` should be exported also? so that you can use it in your own code?

Reply:
Yeah, I think exporting would do no harm. But with this library I have no need to use it hopefully :sweat_smile: